src/bindings/python/src/pyopenvino/core/offline_transformations.cpp

```diff
@@ -105,6 +106,7 @@ void regmodule_offline_transformations(py::module m) {
         "compress_model_transformation",
         [](std::shared_ptr<ov::Model> model) {
             ov::pass::Manager manager;
+            manager.register_pass<ov::pass::DivisionByZeroFP16Resolver>();
             manager.register_pass<ov::pass::MarkPrecisionSensitiveSubgraphs>();
             manager.register_pass<ov::pass::CompressFloatConstants>();
             manager.run_passes(model);
```

Comment:
@mitruska, is it the right order for `DivisionByZeroFP16Resolver` transformation? Could you please contact @pavel-esir who is working ong this pipeline?

Reply:
Sure, I've already contacted him.
Validation details are in the ticket 66045.